ogre_mesh_viewer.py

```diff
@@ -200,6 +205,23 @@ def draw_loading(self):
         ImGui.Text("Loading..            ")
         ImGui.End()
 
+    def set_orientation(self):
```

Comment:
this method only modifes `self.app` therefore it should be moved to `app`

Reply:
![image](https://github.com/OGRECave/ogre-meshviewer/assets/5248597/f9ec84ec-8359-4f75-8834-9838070a76a5)